pkg/validation/spec/schema.go

```diff
@@ -511,3 +516,33 @@ func (s *Schema) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+func (s *Schema) UnmarshalNextJSON(opts jsonv2.UnmarshalOptions, dec *jsonv2.Decoder) error {
+	var x struct {
+		Extensions
+		SchemaProps
+		SwaggerSchemaProps
+	}
+	if err := opts.UnmarshalNext(dec, &x); err != nil {
+		return err
+	}
+
+	_ = x.Ref.fromMap(x.Extensions)
+	_ = x.Schema.fromMap(x.Extensions)
```

Comment:
add a comment explaining why we're ignoring these errors?

Reply:
whoops. adding that back